main.tf

```diff
@@ -120,6 +122,7 @@ module "vault" {
   use_vault      = var.use_vault
   region         = var.region
   enable_acl     = var.enable_acl
+  depends_on     = [module.cluster.eks]
```

Comment:
Why is this needed?

Reply:
The EBS add-on would result in a DEGRADED status. I suspected it was due to vault resources creation contention. I struggled with trying to determine if this was the case but it did seem to occur only when using Vault secrets and not ASM. I could retest this to confirm if it is required if you think it would be helpful.

Reply:
Not sure what I was experiencing earlier but the dependency isn't necessary. Removed the statement and tested on new builds in both Vault and ASM with success.